zebrad/tests/common/get_block_template_rpcs.rs

```diff
@@ -0,0 +1,3 @@
+use super::*;
```

Comment:
Please add a module doc comment to new modules.

Reply:
Added in [`cf4e937`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/cf4e9376e4af691f2dc1183f53b1c95cdb91bcd5)